commons/src/main/scala/org/bitbucket/inkytonik/cooma/Cooma.syntax

```diff
@@ -31,13 +30,10 @@ Expression {paren} =
   | 'Boolean'                                                {BoolT}
```

Comment:
Is there a reason why `Boolean` is still here?

Reply:
Yes, this PR just removes the built-in type names (i.e., types that are defined by the run-time). Boolean can be defined in Cooma itself, so it will be removed by the prelude stuff I'm working on now.